ee/admin-server/public/index.tsx

```diff
@@ -12,7 +12,7 @@ const Entry = () => {
 
 	const panelButton = (
 		<AnchorButton href={'/_panel/'} size={'small'} distinction={'seamless'}>
-			<Icon blueprintIcon={'cog'} />
+			<Icon blueprintIcon="properties" /> List projects
```

Comment:
👎 
there is much more in a panel - project creation, api keys and user management, profile security (password, 2fa). i thing cog icon (maybe with settings label) is more descriptive